<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="hpdesktop" name="LandingPage" tests="2" failures="2" timestamp="11 Jun 2019 12:48:19 GMT" time="10.728" errors="0">
  <testcase name="FillLoginForm" time="1.905" classname="testcases.TestLandingPage">
    <failure type="java.lang.AssertionError" message="expected [https://tpqi-demo.jigsawgroups.work/?] but found [https://tpqi-demo.jigsawgroups.work/login]">
      <![CDATA[java.lang.AssertionError: expected [https://tpqi-demo.jigsawgroups.work/?] but found [https://tpqi-demo.jigsawgroups.work/login]
	at testcases.TestLandingPage.FillLoginForm(TestLandingPage.java:44)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- FillLoginForm -->
  <testcase name="FillLoginForm" time="1.659" classname="testcases.TestLandingPage">
    <failure type="java.lang.AssertionError" message="expected [https://tpqi-demo.jigsawgroups.work/?] but found [https://tpqi-demo.jigsawgroups.work/login]">
      <![CDATA[java.lang.AssertionError: expected [https://tpqi-demo.jigsawgroups.work/?] but found [https://tpqi-demo.jigsawgroups.work/login]
	at testcases.TestLandingPage.FillLoginForm(TestLandingPage.java:44)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- FillLoginForm -->
</testsuite> <!-- LandingPage -->
